AY2021S2-CS2103-T14-1/tp
# Fix room number constraints and UG for room commands

### What this does
- Fixes #207 by updating room validation regex to prevent 00 as valid input for floor number
- Fixes #208 by updating docs to explicitly allow 000 as valid unit number
- Fixes #206 by clearly specifying constraints of room number in both UG and help message
- Fixes #220 by correcting a typo in the command summary portion of the UG
- Fixes #262 by indicating that room types are not case sensitive
- Fixes #222 by showing user friendly room type strings in the GUI

This PR also clearly documents the room family of commands in the UG with relevant links to the parameter summary.

### How to test
1. Try to add a room with room number starting 00
2. See if the error message clearly tells you how to fix your incorrect input
3. See if room types look friendlier
3. Check the docs:
4. cd to `docs/ `folder of the project.
5. Run `bundle exec jekyll serve`.
6. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
7. Navigate to `Command Parameters` and see if documentation for ROOM_NUMBER, ROOM_TYPE and KEYWORD is clear.
8. Navigate to `Room Management` and see if the documentation room family of commands is clear, and that all links are working.
9. Navigate to `Command Summary` and check if `oedit` does not have a `n/` parameter in the example usage.
